Source/HBIOS/md.asm

```diff
@@ -999,36 +996,34 @@ MD_S_SZ	.EQU	$-MD_FERAS_R		; SIZE OF RELOCATABLE CODE BUFFER REQUIRED
 ;          IX POINTS TO DATA TO BE WRITTEN
 ;          A  CONTAINS CURRENT BANK 
 ; ON EXIT  NO STATUS RETURNED
+;          AF' TRASHED

```

Comment:
This may be an issue for some application software.  Some CP/M application software assumes that the OS/BIOS will not touch the alternate register set.